src/LoginCallback.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017-2020Present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
A space is missing somewhere...

Reply:
we can either keep 2020 or present, checked other repos, looks like `Present` is the consistent way.